webapp/src/main/java/com/box/l10n/mojito/okapi/ImportTranslationsFromLocalizedAssetStep.java

```diff
@@ -48,7 +48,9 @@
         REVIEW_NEEDED,
         TRANSLATION_NEEDED,
         APPROVED
-    };
+    }
+
+    ;
```

Comment:
I don't know if this was intended, but it looks really weird.

Reply:
nope, bad intellij auto-indent. Fixed